### ISSUE TITLE
some questions about mixmatch

### ISSUE THREAD

**Issue by Herb2333** [perrying/realistic-ssl-evaluation-pytorch#4]
Thank you for this repo,it help me a lot,but i run the mixmatch method,find that it does‘t have good performance. i want to know the reason abou this .i also tried some other hyperparameters ,but it isn't do well.

**Comment by min9kwak** [comment]
In this repo, the implementation uses MixUp (for only unlabeled data), not MixMatch (both labeled and unlabeled data).
I think that the experimental framework of the original paper is not suitable for MixMatch, because other comparison algorithms only use unlabeled data for SSL loss.
(MixMatch would not make the experimental code nice and neat)

**Comment by Herb2333** [comment]
Thank you very much.I found other MixMatch code and it works nice.




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "min9kwak"<notifications@github.com&gt;; 
发送时间: 2020年10月8日(星期四) 晚上9:57
收件人: "perrying/realistic-ssl-evaluation-pytorch"<realistic-ssl-evaluation-pytorch@noreply.github.com&gt;; 
抄送: "胡彬"<1367317486@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [perrying/realistic-ssl-evaluation-pytorch] some questions about mixmatch (#4)





 
In this repo, the implementation uses MixUp (for only unlabeled data), not MixMatch (both labeled and unlabeled data).
 I think that the experimental framework of the original paper is not suitable for MixMatch, because other comparison algorithms only use unlabeled data for SSL loss.
 (MixMatch would not make the experimental code nice and neat)
